data/routes/projectRoute.js

```diff
@@ -0,0 +1,82 @@
+const router = require('express').Router()
+
+const Projects = require('../helpers/projectModel')
+const Actions = require('../helpers/actionModel')
+
+
+
+router.get('/:id', (req, res) => {
+    Projects.get(req.params.id)
+    .then(project => {
+        if (!project) {
+            res.status(404).json({message: "No project found by that ID!"})
+        } else {
+            res.status(200).json(project)
+        }
+    })
+    .catch(err => {
+        console.log(err)
+        res.status(500).json(err)
+    })
+})
+
+router.post('/', (req, res) => {
+    Projects.insert(req.body)
```

Comment:
are you absolutly 10000000000% positive that req.body is what you want?? i like to console log these out to check.